flashinfer/gdn_kernels/gdn_decode_bf16_wy_output_only.py

```
@@ -0,0 +1,2255 @@
+    # Exception: the strided-qkv opt-in path passes non-compact q/k/v whose
+    # descriptors stay fully static, so it keeps B/pool in the key (fallback).
+    cache_key: tuple = (
+        str(device),
+        mbp,
+        t_disc,
+        n_valid,
+        _qkv_rs,
+        _ab_native_flag,
+    )
+    if _qkv_rs > 0:
+        cache_key = cache_key + (B, h0.shape[0])
```

Ruff (0.15.20)

[warning] Consider `(*cache_key, B, h0.shape[0])` instead of concatenation

Replace with `(*cache_key, B, h0.shape[0])`

(RUF005)